src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+$calendar-day:100px;
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  overflow-x: hidden;
+}
+
+.calendar {
+  display: flex;
+  width: 706px;
+  min-width: 706px;
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    display: flex;
+    width: $calendar-day;
+    height: $calendar-day;
+    border: 1px solid black;
+    background-color: #eee;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: center;
+    transition: all 0.5s;
```

Comment:
please list all properties you want to use transition on - that way it's more readable